docs/cow-protocol/tutorials/solvers/flashloans_support.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 4
+---
+
+# How to execute flashloans orders
+
+This short tutorial explains how orders with flashloans hints can be executed by solvers.
+
+## Where to get information about an order’s flashloan?
+
+This information is added by the user inside the order’s appdata. If you have a colocated driver you’ll only get the appdata hash as part of the order from the autopilot.
```

Comment:
Even at risk of repetition, I think we can be more mindful of people not knowning much about things like the appData.

Orders contain some additional meta-data inside a field called appData (read more here). In particular, the flashloan hint is provided in a metadata with the following information
```
"flashloan": {
      "amount": "20000000000000000000",
      "liquidityProvider": "0xb50201558B00496A145fE76f7424749556E326D8",
      "protocolAdapter": "0x19167A179bcDDBecc2d655d059f20501E5323560",
      "receiver": "0x19167A179bcDDBecc2d655d059f20501E5323560",
      "token": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
    }
 ```
 Check the following order in the explorer and expand the appData field to reveal how this hint is included in the order.